libs/fixtures/src/index.test.ts

```diff
@@ -0,0 +1,18 @@
+import * as fixtures from '.'
+
+test('has various election definitions', () => {
+  expect(Object.keys(fixtures)).toMatchInlineSnapshot(`
+    Array [
+      "electionSample",
+      "primaryElectionSample",
+      "multiPartyPrimaryElection",
+      "electionSampleLongContent",
+      "electionWithMsEitherNeither",
+      "electionSampleDefinition",
+      "primaryElectionSampleDefinition",
+      "multiPartyPrimaryElectionDefinition",
+      "electionSampleLongContentDefinition",
+      "electionWithMsEitherNeitherDefinition",
```

Comment:
I think starting with "sample" could be okay, though I'd argue "example" is better since sample implies statistical sampling which is not what this is.